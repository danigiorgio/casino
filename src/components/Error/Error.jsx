import PropTypes from 'prop-types';

const Error = ({ children }) => {
  return (
    <div className="ui error message">
      <p className="header">{children}</p>
    </div>
  );
};

Error.propTypes = {
  children: PropTypes.node,
};

export default Error;
