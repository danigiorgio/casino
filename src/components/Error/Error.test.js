import { render } from '@testing-library/react';

import Error from './Error';

describe('<Error />', () => {
  it('should render Error', () => {
    const { container } = render(<Error />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
