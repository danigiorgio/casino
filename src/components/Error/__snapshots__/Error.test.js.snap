// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should render Error 1`] = `
<div
  class="ui error message"
>
  <p
    class="header"
  />
</div>
`;
